packages/common/src/resources/string-to-blob.ts

```diff
@@ -5,7 +5,7 @@
  * @param {string} the string
  */
 export function stringToBlob(s: string, type = "application/octet-stream") {
-  /* istanbul ignore next */
+  /* istanbul ignore else -- @preserve */
```

Comment:
`/* instanbul ignore next -- @preserve */` ?

Reply:
yeah, all of these were actually ignoring more LOC that they needed to by improperly using `next` instead of `else` or `if`